nautobot/docs/installation/4-gunicorn.md

```diff
@@ -33,43 +33,102 @@ max_requests_jitter = 500
 ```
 
 This configuration should suffice for most initial installations, you may wish to edit this file to change the bound IP
-address and/or port number, or to make performance-related adjustments. See [the Gunicorn
+address and/or port number, or to make performance-related adjustments. See [Gunicorn
 documentation](https://docs.gunicorn.org/en/stable/configure.html) for the available configuration parameters.
 
-## systemd Setup
+## Setup systemd
 
-We'll use systemd to control both gunicorn and Nautobot's background worker process. First, copy
-`contrib/nautobot.service` and `contrib/nautobot-rq.service` to the `/etc/systemd/system/` directory and reload the
-systemd dameon:
+We'll use `systemd` to control both Gunicorn and Nautobot's background worker process. 
+
+!!! warning
+    The following steps must be performed with root permissions.
+
+### Nautobot service
+
+First, copy and paste the following into `/etc/systemd/system/nautobot.service`:
```

Comment:
Yeah this being an interim step, I'm not opposed to keeping the files there, but I really would like to to self-contain the deployment without having to clone or copy files around. Either by making the installer script more robust, having management commands that can do it, having the docs tell you where to download the files directly, or having something kind of like what Poetry does with `get-poetry.py` (`get-nautobot.py`?).

Reply:
The management-commands approach isn't ideal since these specific operations require `root` rather than `nautobot` user access. I think probably the installer script / get-nautobot script approach would be good though.